server/aws-lsp-codewhisperer/src/language-server/telemetryService.ts

```diff
@@ -132,6 +140,43 @@ export class TelemetryService extends CodeWhispererServiceToken {
     }
 
     public emitUserTriggerDecision(session: CodeWhispererSession, timeSinceLastUserModification?: number) {
+        if (this.enableTelemetryEventsToDestination) {
```

Comment:
We still need explicit unit test for cases when `enableTelemetryEventsToDestination` is false. Currently we don't have test that would verify that this codepath is not called.

Reply:
added the same.